dev/ansible/04_upload_files.yml

```diff
@@ -1,6 +1,31 @@
+- name: Check if intention allow exists for minio-local and minio
+  shell: consul intention match -source minio-local
+  register: intention_exist
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  tags: example-upload
+
+- name: Consul create intention allow
+  shell: nohup consul intention create minio-local minio
+  register: test_output_intention
+  when: intention_exist is defined and intention_exist.rc != 0
+  environment:
+    CONSUL_HTTP_TOKEN: "master"
+  tags: example-upload
+
```

Comment:
I think it is not necessary as long as we rely on tasks running sequence in alphanumeric order.
Means that all necessary intentions must be created/verified before in `dev/ansible/00_create_intention_allow_mc_minio.yml`


```suggestion
```